app/src/main/java/com/example/android/psiak/ui/main/MainPresenter.java

```diff
@@ -13,51 +14,57 @@
  */
 
 class MainPresenter
-        extends BasePresenter<MainContract.View>
-        implements MainContract.Presenter<MainContract.View> , FirebaseDataListener {
+    extends BasePresenter<MainContract.View>
+    implements MainContract.Presenter<MainContract.View>, FirebaseDataListener {
```

Comment:
Presenter shouldn't be coupled with Firebase implementation. Maybe new class that will encapsulate Firebase implementation details?